tests/test_auth_cli.py

```diff
@@ -57,10 +57,9 @@ def mock_stored_tokens():
         "token2": "hf_5678",
         "active_token": "hf_9012",
     }
-    with patch("huggingface_hub._login.get_stored_tokens", return_value=stored_tokens), patch(
-        "huggingface_hub.utils._auth.get_stored_tokens", return_value=stored_tokens
-    ):
-        yield stored_tokens
+    with patch("huggingface_hub._login.get_stored_tokens", return_value=stored_tokens):
+        with patch("huggingface_hub.utils._auth.get_stored_tokens", return_value=stored_tokens):
+            yield stored_tokens
```

Comment:
Less elegant that the previous version but ruff was formatting it in a way that was not supported in Python3.8